src/renderers/dom/shared/ReactDOMComponent.js

```diff
@@ -742,9 +743,11 @@ ReactDOMComponent.Mixin = {
           }
           propValue = CSSPropertyOperations.createMarkupForStyles(propValue, this);
         }
-        var markup = null;
+        var markup = '';
```

Comment:
No idea.  Presumably I wouldn't have made it if it wasn't necessary, but I don't see the reason for it and it passes unit tests as null so I'll go ahead and remove it.